public/js/script.js

```diff
@@ -41,4 +70,5 @@ function loadEnd(event) {
   console.log(fastQCFile);
   linePlot.render(fastQCFile.modules.qual.quintiles);
   boxPlot.render(fastQCFile.modules.qual.quintiles);
+  parseBasicInfo(fastQCFile.modules.basic)
 }
```

Comment:
You're not really parsing anything because the module is already neatly organised as an object. It's following the same model as the D3 plotting functions.. taking a module and putting that info the DOM. I think ``addBasicTable`` would be pretty clear what the function is doing.

And I've realised you don't need to add the ``$('#basic-table-insert').empty()`` function because you're currently using ``replaceWith()`` so ignore that useless comment. :P

It might be worth thinking about what a good way to display multiple tables might be. Maybe just a simple ``dropdown-toggle`` menu above the table?  But can save that for another PR once the multiple file uploader is added.